### PR TITLE
Prepare for next release

### PR DESCRIPTION
**Pull request by oschwald**
If this looks ok, I think we can do a release.

* Removed use of `File::Slurp::Tiny`. We can just use `->slurp` on `Path::Class::File`, which we are already using.
* Silenced `Perl::Critic`
* Set the `copyright_year`

### CODE REVIEW

**Comment by autarch** [comment]
Well, it's failing in Travis.

**Comment by autarch** [comment]
Doh, didn't mean to close it.

**Comment by oschwald** [comment]
The Travis failure was from the `Pod::Coverage` test on a test module. I just ignored it.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -96,8 +96,8 @@ instructions for doing that is available here:
 https://help.github.com/articles/creating-a-pull-request
 
 If you have found a bug, but do not have an accompanying patch to fix it, you
-can submit an issue report [via the web](http://rt.cpan.org/Public/Dist/Display.html?Name=Database-Migrator)
-or [via email](bug-database-migrator@rt.cpan.org.
+can submit an issue report [via the web](WARNING: bugtracker data not set!)
```
Comment (klp2) [inline CONTRIBUTING.md]: Looks like the link to github isn't getting set.

Comment (oschwald) [inline CONTRIBUTING.md]: Nice catch.

Comment (oschwald) [inline CONTRIBUTING.md]: This was because the repo had issues disabled. I enabled them and regenerated.

**Pull request merged**